### PR TITLE
[RU]Grouple domain switcher

### PR DESCRIPTION
**Pull request by e-shl**
Closes https://github.com/tachiyomiorg/tachiyomi-extensions/issues/17924
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by e-shl** [comment]
Selfmanga a new domain exists, but the functionality of the directory is broken on the site itself. The old domain retains full functionality via VPN outside of Russia.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt [hunk 1]
```diff
@@ -9,12 +9,12 @@ class GroupLeGenerator : ThemeSourceGenerator {
...
     override val themeClass = "GroupLe"
 
-    override val baseVersionCode = 19
+    override val baseVersionCode = 20
 
     override val sources = listOf(
         SingleLang("ReadManga", "https://readmanga.live", "ru", overrideVersionCode = 46),
-        SingleLang("MintManga", "https://mintmanga.live", "ru", overrideVersionCode = 46),
-        SingleLang("AllHentai", "https://2023.allhen.online", "ru", isNsfw = true, overrideVersionCode = 22),
+        SingleLang("MintManga", "https://mintmanga.com", "ru", overrideVersionCode = 46),
```
Comment (vetleledaal) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt:16]:
```suggestion
        SingleLang("MintManga", "https://mintmanga.com", "ru", isNsfw = true, overrideVersionCode = 46),
```
https://mintmanga.com/list/limitation/r18_

**Review by vetleledaal — COMMENTED** [review]
https://mintmanga.live seems to be online still, I guess you changed it for a reason, instead of adding another site. I don't speak the language, so it's mostly up to you

**Comment by e-shl** [comment]
> https://mintmanga.live seems to be online still, I guess you changed it for a reason, instead of adding another site. I don't speak the language, so it's mostly up to you

blocking/censorship

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt (hunk 1 above)
Comment (e-shl) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt:16]: No Hentai source.

Comment (vetleledaal) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt:16]: even if it's not mainly hentai, it's still NSFW https://mintmanga.com/persikovyi_sorbet__A35c96/vol1/3?mtr=1

**Pull request merged**